drivers/volume/portworx/schedops/schedops.go

```diff
@@ -11,6 +11,10 @@ import (
 
 // Driver is the interface for portworx operations under various schedulers
 type Driver interface {
+	// EnableOnNode enable portworx on given node
```

Comment:
can you fix the function comments for start and stop? They are still called enable and disable